spec/tests/resources.js

```diff
@@ -74,7 +74,7 @@ test('resources:', function (t) {
   t.test('setup', function (setup) {
     db.collections.create(testCollection)
       .then(function () {
-        return db.documents.upload(new Buffer('1'))
+        return db.documents.upload(Buffer.alloc(1))
```

Comment:
I think this also needs to be `buffer.from('1')`

Reply:
Yeah, otherwise we would not be uploading a file that consists of a `1`, which is a valid query that does nothing.